server/auth/github/github.go

```diff
@@ -22,12 +22,17 @@ const (
 type Config struct {
 	config       *oauth2.Config
 	organization string
+	whitelist    map[string]bool
 }
 
 // New creates a new Github provider from a configuration.
 func New(c *config.Auth) (auth.Provider, error) {
-	if c.ProviderOpts["organization"] == "" {
-		return nil, errors.New("github_opts organization must not be empty")
+	users_whitelist := make(map[string]bool)
```

Comment:
Or better yet, just use something short like `uw`. It's immediately obvious what it is from the next line.